src/rpm/package.rs

```diff
@@ -101,7 +99,7 @@ impl RPMPackage {
 
     /// Create package signatures using an external key and add them to the signature header
     #[cfg(feature = "signature-meta")]
-    pub fn sign<S>(&mut self, signer: S) -> Result<(), RPMError>
+    pub fn sign<S>(&mut self, signer: S, t: Timestamp) -> Result<(), RPMError>
```

Comment:
Add an example here

Reply:
Hm, FYI none of the other `RPMPackage` methods contain examples. Should I construct `RPMPacakge` in example using the builder?

Reply:
You can just copy a minimal part of the example, just enough to demonstrate signing and the various ways you could construct a `Timestamp`.

Reply:
Also maybe instead of changing the entire signature it could be two functions, where `sign()` just calls `sign_with_timestamp(key, Timestamp::now())`

Reply:
Done.